devel/autoconf269/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            autoconf269
```

Comment:
your commit message should likely be: `autoconf269: new port`?

Reply:
@reneeotten indeed!